build/docker/copy-test-operations.sh

```diff
@@ -20,5 +20,7 @@ echo "Copying test artifacts to install location..."
 find ${WORKSPACE}/conformance -iname 'fhir-ig*.jar' -not -iname 'fhir*-tests.jar' -not -iname 'fhir*-test-*.jar' -exec cp -f {} ${USERLIB} \;
 cp -pr ${WORKSPACE}/operation/fhir-operation-test/target/fhir-operation-*-tests.jar ${USERLIB}
 cp -pr ${WORKSPACE}/operation/fhir-operation-term-cache/target/fhir-operation-*.jar ${USERLIB}
+cp -pr ${WORKSPACE}/operation/fhir-operation-cpg/target/fhir-operation-*-shaded.jar ${USERLIB}
+cp -pr ${WORKSPACE}/operation/fhir-operation-cqf/target/fhir-operation-*-shaded.jar ${USERLIB}
```

Comment:
Let's keep this separate, and not add the operation here. 
I'd like to create a separate workflow for it, and I can help you do that.

It'd also require you add some logic to any IT tests to check the CapabilityStatement (CarinBlueButtonTest is one example where it uses it's AbstractProfileTest to see if the IG is installed)

Reply:
I don't think you need any automation changes. and I'd prefer not to have them. 
We can setup automation specific to integration testing this feature at a later date